### DISCUSSION TITLE
Feature Request: Light Slider: Tap to quickly turn on/off, long press for options

### DISCUSSION THREAD

**Discussion by Coder84619** [Clooos/Bubble-Card#324] [Questions about config, custom styles and templates] [answered]
In Apple Home I really like the ability to simply tap a light to immediately turn it on or off. To choose color temp or set other options you long press on the button. I'd like the same feature set for the light 'slider' bubble card entity. 

As it works now I can tap on the bubble slider to turn a light on, but I haven't found a way to single tap on the slider to turn it off. Taping on the light icon should turn the light off, while taping anywhere else on the slider should stay the same (setting the light level to the tapped position). Then a LONG press should bring up the floating window to bring up the options interface.

**Comment by Clooos** [comment, marked as answer]
Hi and thanks for your feedback! You can do that with tap actions, take a look at this part of the documentation:

https://github.com/Clooos/Bubble-Card#tap-double-tap-and-hold-actions

Then just add this to your cards:
```yaml
tap_action: 
  action: toggle
hold_action:
  action: more-info
```
